lib/buffer.js

```diff
@@ -1178,7 +1180,7 @@ Buffer.prototype.readDoubleBE = function readDoubleBE(offset, noAssert) {
 
 function checkInt(buffer, value, offset, ext, max, min) {
   if (value > max || value < min)
-    throw new TypeError('"value" argument is out of bounds');
+    throw new errors.TypeError('ERR_INVALID_OPT_VALUE', 'value', value);
```

Comment:
I don't know if this error code is suitable.
Should we have a new error code for it?